### ISSUE TITLE
Klipper with Arduino Due Connection failure

### ISSUE THREAD

**Issue by bruce356** [KevinOConnor/klipper#844]
hi, trying to get Klipper to work with DUE and RAMPS-FD v2.1a.
Have flashed the Due using programming port (ttyUSB0) that was successful (using ttyACM0 failed).

Connecting through programming port or Native port both failed.
Hoping someone can assist.
Thanks and regards - bruce
[klippy_DUE.zip](https://github.com/KevinOConnor/klipper/files/2541835/klippy_DUE.zip)
[Octoprint Klipper Not Conect DUE Log.zip](https://github.com/KevinOConnor/klipper/files/2541838/Octoprint.Klipper.Not.Conect.DUE.Log.zip)

**Comment by jakep82** [comment]
If you flashed at /dev/ttyUSB0, then you also need to use that in your config.  

https://github.com/KevinOConnor/klipper/blob/master/docs/FAQ.md#wheres-my-serial-port

Also, if you're connecting at ttyUSB0, that would imply you're using a clone Due board.  Virtually all clone boards available have a hardware issue that prevents them from running code until you press the reset button.  It might be possible to fix, but it will likely require a modification to the Due.  I got tired of dealing with it and bought a genuine Due which doesn't have this problem.

https://forum.arduino.cc/index.php?topic=256771.60

**Comment by bruce356** [comment]
Hi @jakep82 , much appreciate your help, yes you are right its a clone, I did try to connect using ttyUSB0 but that was not working nor were any others I tried. 
` Virtually all clone boards available have a hardware issue that prevents them from running code until you press the reset button.`

Thanks for the advice, once reset is pressed using USB0 it connects without problem.
Tried also connecting to the native USB port but without luck.
`ls -l /dev/serial/by-id/`     does not work for me as I do not have  a serial folder in octoprint.

Regards - bruce
